### PR TITLE
Eliminate misleading dependence on voxel size from functions

### PR DESCRIPTION
**Pull request by paolafer**
I've realized that we kept the explicit dependence on voxel size in make_track_graph function, when we added the size property to Voxel. This PR fixes this.

### CODE REVIEW

**Comment by jacg** [comment]
This would have been a nice exercise for @carmenromo ...

**Comment by paolafer** [comment]
She did the same for the merge_tracks function.

**Review by jjgomezcadenas — APPROVED** [review]
Simple and clear.

**Comment by gonzaponte** [comment]
@paolafer After merging #423 this branch has conflicts. Solve them, please.

**Comment by jacg** [comment]
> this branch has conflicts

They look pretty simple. If you could work through the process with @carmenromo, that would be good.

**Comment by paolafer** [comment]
I’m traveling right now and I won’t be back until Sunday. I don’t know if a Skype-peras-conflict-solving is worth the effort in this case...

**Comment by jacg** [comment]
No, no point in doing it over Skype, this needs genuine peras (I could have done it myself, if that were an option). If it's urgent to get this done then just do it yourself. If it can wait until you get back, then I would recommend waiting. As the conflict is tiny, it's easy to understand in its entirity. As @carmenromo is familiar with this code, it should make some sort of sense, rather than being an abstract exercise, and it would serve as a good introduction to both conflict resolution and rebasing code within a PR.

In short, it's a good didactic opportunity which we should take advantage of, if we can afford to.

**Comment by paolafer** [comment]
I understood that @jjgomezcadenas wanted the more PR's approved as soon as possible, correct me if I'm wrong. @carmenromo and myself are working on the merging algorithm, which will offer opportunities of learning about rebasing and (maybe) conflict resolution, too.

**Comment by jacg** [comment]
Repetition and good guidance are vital to learning. I doubt that this PR is holding anything else back. So I would recommend using it as a training opportunity. However, if you or @jjgomezcadenas think otherwise, then just go ahead and merge it.
